File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/2052/BusinessApplication/Controls/VisualStates.cs

namespace $safeprojectname$.Controls
{
    /// <summary>
    /// 控件中的可视状态的名称和帮助程序。
    /// </summary>
    internal static class VisualStates
    {
        #region GroupBusyStatus
        /// <summary>
        /// BusyIndicator 的 Busy 状态。
        /// </summary>
        public const string StateBusy = "Busy";

        /// <summary>
        /// BusyIndicator 的 Idle 状态。
        /// </summary>
        public const string StateIdle = "Idle";

        /// <summary>
        /// 繁忙状态组名称。
        /// </summary>
        public const string GroupBusyStatus = "BusyStatusStates";
        #endregion

        #region GroupVisibility
        /// <summary>
        /// BusyIndicator 的 Visible 状态名称。
        /// </summary>
        public const string StateVisible = "Visible";

        /// <summary>
        /// BusyIndicator 的 Hidden 状态名称。
        /// </summary>
        public const string StateHidden = "Hidden";
        
        /// <summary>
        /// BusyDisplay 组。
        /// </summary>
        public const string GroupVisibility = "VisibilityStates";
        #endregion
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/1049/BusinessApplication/Controls/VisualStates.vb

Namespace Controls
    ''' <summary>
    ''' Имена и помощники для визуальных состояний в элементах управления.
    ''' </summary>
    Friend Module VisualStates

#Region "GroupBusyStatus"
        ''' <summary>
        ''' Состояние занятости (Busy) для элемента BusyIndicator.
        ''' </summary>
        Public Const StateBusy As String = "Busy"

        ''' <summary>
        ''' Неактивное состояние (Idle) для элемента BusyIndicator.
        ''' </summary>
        Public Const StateIdle As String = "Idle"

        ''' <summary>
        ''' Имя группы состояний занятости.
        ''' </summary>
        Public Const GroupBusyStatus As String = "BusyStatusStates"
#End Region

#Region "GroupVisibility"
        ''' <summary>
        ''' Имя состояния видимости Visible для элемента BusyIndicator.
        ''' </summary>
        Public Const StateVisible As String = "Visible"

        ''' <summary>
        ''' Имя состояния невидимости (Hidden) для элемента BusyIndicator.
        ''' </summary>
        Public Const StateHidden As String = "Hidden"

        ''' <summary>
        ''' Группа BusyDisplay.
        ''' </summary>
        Public Const GroupVisibility As String = "VisibilityStates"
#End Region
    End Module
End Namespace